pages/api/account/statistics.js

```diff
@@ -24,9 +24,19 @@ export default async function handler(req, res) {
   res.status(200).json(data);
 }
 
-export async function getStats(username) {
+export async function getStats(username, numberOfDays) {
   await connectMongo();
 
+  // is the format okay? or should it be in this => '1m, 2m, 1y, 4m' instead of the numbers
+  if(numberOfDays === null || numberOfDays === undefined || numberOfDays > 365){
+    numberOfDays = 30
+  }
```

Comment:
I think if we default on the function parameters, then these checks are not needed

Reply:
Absolutely, you are right👍🏽 totally forgot about the default